modules/apps/object/object-web/src/main/resources/META-INF/resources/js/tests/ObjectDetails/ObjectDataContainer.spec.tsx

```diff
@@ -0,0 +1,174 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+import {render, screen} from '@testing-library/react';
+import React from 'react';
+
+import {ObjectDataContainer} from '../../components/ObjectDetails/ObjectDataContainer';
+
+const baseValuesMock = {
+	active: false,
+	defaultLanguageId: 'en_US',
+	externalReferenceCode: '',
+	id: 0,
+	label: {en_US: 'Label Test'},
+	modifiable: false,
+	name: 'Object Name',
+	pluralLabel: {en_US: 'Plural Label Test'},
+	system: false,
+	titleObjectFieldName: '',
+} as Partial<ObjectDefinition>;
+
+describe('ObjectDataContainer component', () => {
+	it('check whether the object name input value is displayed correctly and is required', () => {
+		render(
```

Comment:
To keep the present simple we could do something like:

```suggestion
	it('shows the object definition name input value and makes it required', () => {
		render(
```